Update docs to note schedule action delay (#3059) (#3060)
github/docs: content/actions/reference/events-that-trigger-workflows.md

The `schedule` event allows you to trigger a workflow at a scheduled time.

{% note %}

> Note: Currently due to load `schedule` actions can have up to an hour delay, this is being looked into but please make considerations for this in the interim.

Reviewer: @BrentonWheeler Thank you for your patience while I discussed this with the engineers! Here is a suggested change after discussing with them:

```suggestion
Note: Due to load, the `schedule` event may be delayed 
```

Reviewer: Since you and others have been waiting for this update, I'm going to go ahead and incorporate the suggested change and get this merged down for you.
